models.py

```diff
@@ -95,7 +95,20 @@ class ModelArgs:
     audio_num_codebooks: int
 
 
-class Model(nn.Module):
+class Model(
+    nn.Module,
+    PyTorchModelHubMixin,
+    repo_url="https://github.com/SesameAILabs/csm",
+    pipeline_tag="text-to-speech",
+    license="apache-2.0",
+    coders={
```

Comment:
Btw one more remark, this "coders" argument was necessaary since the first keyword argument of the Model class is called "args". If you rename it to "config", then this "coders" thing can be removed.

We made a fix in https://github.com/huggingface/huggingface_hub/pull/2928 so that the first keyword argument does not have be called "config", but that will be part of the next PyPi release.

Reply:
That's a lot cleaner. I've updated the model HF repo. If you remove coders here we can merge

You can copy the [Model changes from here](https://huggingface.co/spaces/sesame/csm-1b/commit/1c1de51030ddfb02a472733f31ed231bda99b1a0)

Reply:
Great, I pushed a change.